connectors/sources/s3.py

```diff
@@ -185,19 +186,23 @@ async def get_docs(self):
                         doc_id = md5(
                             f"{bucket}/{obj_summary.key}".encode("utf8")
                         ).hexdigest()
+                        object_size = await obj_summary.size
 
                         doc = {
                             "_id": doc_id,
                             "filename": obj_summary.key,
-                            "size": await obj_summary.size,
+                            "size": f"{object_size} bytes",
```

Comment:
this turns the field into a string, is this what we really want @ppf2 ?

Reply:
Certainly not :) We don't want to change the field into a string.  `size` should remain an integer/long field to support sorting and mathematical functions.  We simply need to provide an indicator of the unit of measure, not to change the actual value of the field (e.g. either update the field name to include bytes there, or add a tooltip to the UI for `size` to tell the user that it is in bytes, etc..

Reply:
If we are planning to a tooltip for this so it is going to be handle from the UI level and may be @tarekziade can handle this thing but if we have to update field name to include bytes in that case can you please suggest what's the most suitable name for this.
example: **size (in bytes)**, **size (bytes)**

Reply:
Please let us know if you want me to update the size key name to size (bytes) or anything else. However if we are planning to add a tooltip, It needs to be taken care at the UI level.
cc Tarek

Reply:
+@danajuratoni This feels like a PM decision on the design.

Reply:
Here's my delayed (sorry for the delay) comment on wether "bytes" should be in the field name (to include bytes) or a tooltip in UI. 
Atm the only feasible option is to change the field name to include bytes, there is no tooltip capability in 8.7. We'll want to add that in 8.8, but for now we should use the field name.

Reply:
I have updated the PR